adafruit_ht16k33/segments.py

```diff
@@ -32,123 +32,220 @@
-    0b00100011, 0b01011101, # &
-    0b00000100, 0b00000000, # '
-    0b00100100, 0b00000000, # (
-    0b00001001, 0b00000000, # )
-    0b00111111, 0b11000000, # *
-    0b00010010, 0b11000000, # +
-    0b00001000, 0b00000000, # ,
-    0b00000000, 0b11000000, # -
-    0b00000000, 0b00000000, # .
-    0b00001100, 0b00000000, # /
-    0b00001100, 0b00111111, # 0
-    0b00000000, 0b00000110, # 1
-    0b00000000, 0b11011011, # 2
-    0b00000000, 0b10001111, # 3
-    0b00000000, 0b11100110, # 4
-    0b00100000, 0b01101001, # 5
-    0b00000000, 0b11111101, # 6
-    0b00000000, 0b00000111, # 7
-    0b00000000, 0b11111111, # 8
-    0b00000000, 0b11101111, # 9
-    0b00010010, 0b00000000, # :
-    0b00001010, 0b00000000, # ;
-    0b00100100, 0b01000000, # <
-    0b00000000, 0b11001000, # =
-    0b00001001, 0b10000000, # >
-    0b01100000, 0b10100011, # ?
-    0b00000010, 0b10111011, # @
-    0b00000000, 0b11110111, # A
-    0b00010010, 0b10001111, # B
-    0b00000000, 0b00111001, # C
-    0b00010010, 0b00001111, # D
-    0b00000000, 0b11111001, # E
-    0b00000000, 0b01110001, # F
-    0b00000000, 0b10111101, # G
-    0b00000000, 0b11110110, # H
-    0b00010010, 0b00000000, # I
-    0b00000000, 0b00011110, # J
-    0b00100100, 0b01110000, # K
-    0b00000000, 0b00111000, # L
-    0b00000101, 0b00110110, # M
-    0b00100001, 0b00110110, # N
-    0b00000000, 0b00111111, # O
-    0b00000000, 0b11110011, # P
-    0b00100000, 0b00111111, # Q
-    0b00100000, 0b11110011, # R
-    0b00000000, 0b11101101, # S
-    0b00010010, 0b00000001, # T
-    0b00000000, 0b00111110, # U
-    0b00001100, 0b00110000, # V
-    0b00101000, 0b00110110, # W
-    0b00101101, 0b00000000, # X
-    0b00010101, 0b00000000, # Y
-    0b00001100, 0b00001001, # Z
-    0b00000000, 0b00111001, # [
-    0b00100001, 0b00000000, # \
-    0b00000000, 0b00001111, # ]
-    0b00001100, 0b00000011, # ^
-    0b00000000, 0b00001000, # _
-    0b00000001, 0b00000000, # `
-    0b00010000, 0b01011000, # a
-    0b00100000, 0b01111000, # b
-    0b00000000, 0b11011000, # c
-    0b00001000, 0b10001110, # d
-    0b00001000, 0b01011000, # e
-    0b00000000, 0b01110001, # f
-    0b00000100, 0b10001110, # g
-    0b00010000, 0b01110000, # h
-    0b00010000, 0b00000000, # i
-    0b00000000, 0b00001110, # j
-    0b00110110, 0b00000000, # k
-    0b00000000, 0b00110000, # l
-    0b00010000, 0b11010100, # m
-    0b00010000, 0b01010000, # n
-    0b00000000, 0b11011100, # o
-    0b00000001, 0b01110000, # p
-    0b00000100, 0b10000110, # q
-    0b00000000, 0b01010000, # r
-    0b00100000, 0b10001000, # s
-    0b00000000, 0b01111000, # t
-    0b00000000, 0b00011100, # u
-    0b00100000, 0b00000100, # v
-    0b00101000, 0b00010100, # w
-    0b00101000, 0b11000000, # x
-    0b00100000, 0b00001100, # y
-    0b00001000, 0b01001000, # z
-    0b00001001, 0b01001001, # {
-    0b00010010, 0b00000000, # |
-    0b00100100, 0b10001001, # }
-    0b00000101, 0b00100000, # ~
-    0b00111111, 0b11111111,
+    0b00000000,
```

Comment:
Please disable Black formatting on this code block and return it to its original format. Thank you.